<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublishItemWidget</class>
 <widget class="QWidget" name="PublishItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PublishItemWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget_progress_image{
	border: 1px solid #000;
	border-top-left-radius:10px;
	border-bottom-left-radius:10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_center" native="true">
     <property name="minimumSize">
      <size>
       <width>248</width>
       <height>204</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>300</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_publish_info" native="true">
        <property name="minimumSize">
         <size>
          <width>235</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="verticalWidget" native="true">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_image">
              <property name="minimumSize">
               <size>
                <width>248</width>
                <height>160</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>200</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../themes/green/green.qrc">:/images/view/image@2x.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_scene">
              <property name="minimumSize">
               <size>
                <width>123</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>160</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_reason">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../themes/green/green.qrc">:/images/view/icon_expired_normal.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnDetail">
              <property name="minimumSize">
               <size>
                <width>54</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>54</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string>查看</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="verticalWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_publishtime">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../themes/green/green.qrc"/>
 </resources>
 <connections/>
</ui>
